src/ray/raylet/worker_pool.cc

```diff
@@ -309,6 +309,7 @@ std::tuple<Process, StartupToken> WorkerPool::StartWorkerProcess(
                       std::to_string(worker_startup_token_counter_));
     options.push_back("-Dray.internal.runtime-env-hash=" +
                       std::to_string(runtime_env_hash));
+    options.push_back("-Dray.job.id=" + job_id.Hex());
```

Comment:
No need

Reply:
This is necessary.  Because this PR enable the eager initialization for function manager, which needs to load classes from local file by the concatenated file path like `/tmp/job/0100000/package/xxx`.

So the java worker should know the job id when started.

Reply:
Since job id could be removed from function manager, this command line arg could be removed either.